### PR TITLE
Updated Readmes with the new repositories

### PR DESCRIPTION
**Pull request by sbalint98**
Updated Readmes with the new repositories, and added scripts for repository creation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,17 +66,19 @@ hipSYCL is still in an early stage of development. It can successfully execute m
...
 * `hipSYCL` - contains the actual hipSYCL libraries, tools and headers
 * `hipSYCL-base` - contains the LLVM/clang stack used by hipSYCL. Installation of this package is mandatory.
 * `hipSYCL-rocm` - contains a ROCm stack. This package is only required if you wish to target AMD ROCm GPUs.
-* `hipSYCL-cuda` - provides a CUDA stack. This package is only required if you wish to target CUDA GPUs. **Note: For legal reasons, we do not redistribute the hipSYCL-cuda pacakge**. You will either have to create a CUDA package using `install/scripts/packaging/make-<distribution>-cuda-pkg.sh` or you can install CUDA directly using the `install/scripts/install-cuda.sh` script.
 
-For more information on packaging and how to create hipSYCL packages yourself, please see the [documentation](install/scripts/README.md).
+**Note: For legal reasons, we do not redistribute the hipSYCL-cuda pacakge** This package is only required if you wish to target CUDA GPUs. . You will either have to create a CUDA package using `install/scripts/packaging/make-<distribution>-cuda-pkg.sh` or you can install CUDA directly using the `install/scripts/install-cuda.sh` script.
```
Comment (illuhad) [inline README.md]:
```suggestion
**Note: For legal reasons, we do not redistribute the hipSYCL-cuda pacakge** This package is only required if you wish to target CUDA GPUs. You will either have to create a CUDA package using `install/scripts/packaging/make-<distribution>-cuda-pkg.sh` or you can install CUDA directly using the `install/scripts/install-cuda.sh` script.
```

File: devops/repos/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# hipSYCL repo creation scripts
+
+These scripts create the repositories for the supported distributions
+
+## Operation
+
+The new packages should be placed in the new_pkg_<distribution> folder of the repository, after that the create_all.sh script will move them to the repositories and sign them. if you want to have the repositories elsewhere, you can use the `REPO_BASE_DIR` environmental variable
```
Comment (illuhad) [inline devops/repos/README.md]:
```suggestion
The new packages should be placed in the `new_pkg_<distribution>` folder of the repository, after that the create_all.sh script will move them to the repositories and sign them. if you want to have the repositories elsewhere, you can use the `REPO_BASE_DIR` environmental variable
```

File: devops/repos/README.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+## Signing
+
+The signing is carried out by the default private key of the user executing the create_all.sh.
```
Comment (illuhad) [inline devops/repos/README.md]:
```suggestion
The signing is carried out by the default private key of the user executing the `create_all.sh` script. 
```

**Comment by illuhad** [comment]
Thank you! :)

File: README.md [hunk 4]
```diff
@@ -66,17 +66,19 @@ hipSYCL is still in an early stage of development. It can successfully execute m
...
 * `hipSYCL` - contains the actual hipSYCL libraries, tools and headers
 * `hipSYCL-base` - contains the LLVM/clang stack used by hipSYCL. Installation of this package is mandatory.
 * `hipSYCL-rocm` - contains a ROCm stack. This package is only required if you wish to target AMD ROCm GPUs.
-* `hipSYCL-cuda` - provides a CUDA stack. This package is only required if you wish to target CUDA GPUs. **Note: For legal reasons, we do not redistribute the hipSYCL-cuda pacakge**. You will either have to create a CUDA package using `install/scripts/packaging/make-<distribution>-cuda-pkg.sh` or you can install CUDA directly using the `install/scripts/install-cuda.sh` script.
 
-For more information on packaging and how to create hipSYCL packages yourself, please see the [documentation](install/scripts/README.md).
+**Note: For legal reasons, we do not redistribute the hipSYCL-cuda pacakge** This package is only required if you wish to target CUDA GPUs. You will either have to create a CUDA package using `install/scripts/packaging/make-<distribution>-cuda-pkg.sh` or you can install CUDA directly using the `install/scripts/install-cuda.sh` script.
```
Comment (keryell) [inline README.md]: `pacakge` → `package`

Comment (illuhad) [reply README.md]: Good catch, @sbalint98, can you please fix this typo? :)

**Pull request merged**